project/deps.sc

```diff
@@ -36,42 +36,45 @@ object Deps {
   def ammoniteRepl(sv: String)     =
     if (sv.startsWith("2.")) ivy"com.lihaoyi:::ammonite-repl:${Versions.ammonite}"
-    else ivy"com.lihaoyi:ammonite-cross-23-repl_${ScalaVersions.cross2_3Version}:${Versions.ammonite}"
+    //else ivy"com.lihaoyi:ammonite-cross-23-repl_${ScalaVersions.cross2_3Version}:${Versions.ammonite}"
+    else ivy"com.lihaoyi:ammonite-cross-3.1.1-repl_${ScalaVersions.cross2_3Version}:${Versions.ammonite}"
   def ammoniteReplApi(sv: String)  =
     if (sv.startsWith("2.")) ivy"com.lihaoyi:::ammonite-repl-api:${Versions.ammonite}"
-    else ivy"com.lihaoyi:ammonite-cross-23-repl-api_${ScalaVersions.cross2_3Version}:${Versions.ammonite}"
+    //else ivy"com.lihaoyi:ammonite-cross-23-repl-api_${ScalaVersions.cross2_3Version}:${Versions.ammonite}"
+    else ivy"com.lihaoyi:ammonite-cross-3.1.1-repl-api_${ScalaVersions.cross2_3Version}:${Versions.ammonite}"
   def ammoniteSpark            = ivy"sh.almond::ammonite-spark:0.12.0"
+  def osLib                    = ivy"com.lihaoyi::os-lib:0.8.1"
   def caseAppAnnotations       = ivy"com.github.alexarchambault::case-app-annotations:${Versions.caseApp}"
   def caseApp                  = ivy"com.github.alexarchambault::case-app:${Versions.caseApp}"
-  def collectionCompat         = ivy"org.scala-lang.modules::scala-collection-compat:2.5.0"
+  def collectionCompat         = ivy"org.scala-lang.modules::scala-collection-compat:2.6.0"
   def coursier                 = ivy"io.get-coursier::coursier:2.0.14"
   def coursierApi              = ivy"io.get-coursier:interface:1.0.4"
   def directories              = ivy"io.github.soc:directories:12"
-  def fs2                      = ivy"co.fs2::fs2-core:2.5.9"
+  def fs2                      = ivy"co.fs2::fs2-core:2.5.10"
   def jansi                    = ivy"org.fusesource.jansi:jansi:1.18"
   def jeromq                   = ivy"org.zeromq:jeromq:0.5.2"
   def jsoniterScalaCore        = ivy"com.github.plokhotnyuk.jsoniter-scala::jsoniter-scala-core:${Versions.jsoniterScala}"
   def jsoniterScalaMacros      = ivy"com.github.plokhotnyuk.jsoniter-scala::jsoniter-scala-macros:${Versions.jsoniterScala}"
   def jvmRepr                  = ivy"com.github.jupyter:jvm-repr:0.4.0"
-  def mdoc                     = ivy"org.scalameta::mdoc:2.2.21"
+  def mdoc                     = ivy"org.scalameta::mdoc:2.3.1"
   def metabrowseServer         = ivy"org.scalameta::metabrowse-server:0.2.3"
   def scalafmtDynamic          = ivy"org.scalameta::scalafmt-dynamic:${Versions.scalafmt}"
   def scalaReflect(sv: String) = ivy"org.scala-lang:scala-reflect:$sv"
   def scalaRx                  = ivy"com.lihaoyi::scalarx:0.4.3"
-  def scalatags                = ivy"com.lihaoyi::scalatags:0.9.4"
-  def slf4jNop                 = ivy"org.slf4j:slf4j-nop:1.8.0-beta4"
+  def scalatags                = ivy"com.lihaoyi::scalatags:0.11.1"
+  def slf4jNop                 = ivy"org.slf4j:slf4j-nop:2.0.0-alpha6"
```

Comment:
This is upgrading slf4j to `2.x`. Shouldn't we stick to `1.x` instead?

Reply:
Had issue with messages regarding not finding the slf4jNop. Also note that slf4j had a security vulnerability in 2.x but don't think it is this version. Seems to run ok. Is this whats blocking its merge?

Reply:
Ok. Will have to update and push again.